src/scenes/Home/index.tsx

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
+import { Button } from '@material-ui/core';
+import React, { useState } from 'react';
 import { useSession } from '../../components/Session';
 import { useTitle } from '../../components/title';
+import { CreateOrganization } from '../Organizations/Create';
 
 export const Home = () => {
   useTitle('Home');
   const [session] = useSession();
+  const [open, setOpen] = useState(false);
 
-  return <div>Welcome, {session?.displayFirstName.value ?? 'Friend'}</div>;
+  return (
+    <div>
+      <div>Welcome, {session?.displayFirstName.value ?? 'Friend'}</div>
+      <Button onClick={() => setOpen(true)} color="primary" variant="contained">
+        CreateOrg
+      </Button>
+      <CreateOrganization
+        open={open}
+        onClose={(open: boolean) => {
+          setOpen(open);
+        }}
```

Comment:
```suggestion
        onClose={() => {
          setOpen(false);
        }}
```

Reply:
We should leave this because we can't close dialog during processing create org.

Reply:
That's true. But this change doesn't prevent that.

For that you need to set `disableBackdropClick` on `Dialog` to true will submitting. And also disable the cancel button.

Reply:
Actually, first one I did can prevent that. BTW your option looks better. I will change it